src/assets/app.extensions.json

```diff
@@ -806,7 +861,7 @@
         },
         {
           "id": "app.viewer.share",
```

Comment:
should this be app.toolbar.share ?

Reply:
SHARE_EDIT action has the id: app.toolbar.share.edit

Reply:
@DenysVuika we need  'id' naming conventions 
@adinapitul what would be easy for testing in terms of cheap locators? having app.[page].toolbar.[action] or app.[page].[action]

Reply:
@adinapitul @pionnegru don't forget that `id` values can also be used in testing, we set "element.id" properties to the corresponding id values at runtime

Reply:
precisely, thats why I've asked whats more convenient for id and testing

Reply:
from automation perspective, doesn't matter what the id is exactly, but it would be nice for an action from the toolbar to have the same id on all the toolbars

Reply:
they have different IDs so that developers can address them separately, i.e. removing buttons from the Viewer toolbar, but leaving in the main page, and vice versa

Reply:
In the custom extension, on could disable button like:

```
{
  "id": "app.viewer.share",
  "disabled": true
}
```

Reply:
then the SHARE_EDIT action from the viewer should follow the same logic and be called app.viewer.share.edit maybe